bot/src/players/players.spec.ts

```diff
@@ -0,0 +1,21 @@
+import { PlayersService } from './players.service';
+import { SoundPlayer } from './sound.player';
+
+
+describe("get method test", () => {
+    let playersService: PlayersService;
+    let new_i: SoundPlayer | undefined;
+    let serv_i: SoundPlayer | undefined;
+    
+    playersService = new PlayersService();
+    new_i = new SoundPlayer('test');
+    serv_i = playersService.get('test');
+
+    test("Should initiate new instance: ", () => {
+        expect(new_i).toStrictEqual(serv_i);
```

Comment:
you should use https://jestjs.io/docs/jest-object#jestspyonobject-methodname to check if constructor was called